rcsb/utils/dictionary/DictMethodEntityHelper.py

```diff
@@ -2641,8 +2614,65 @@ def buildEntityAnnotations(self, dataContainer, catName, **kwargs):
                             cObj.setValue(str(self.__pfP.getVersion()), "assignment_version", ii)
                             #
                             ii += 1
+            # --- CARD annotations
+            if self.__cardP:
+                eTypeD = self.__commonU.getEntityTypes(dataContainer)
+                for entityId, eType in eTypeD.items():
+                    if eType not in ["polymer", "branched"]:
```

Comment:
@piehld Do `branched` entities have CARD annotations?

Reply:
@brindakv Good point—no, they don't appear to have any. I copied that part from John's code for assigning CARD features, so I'm not sure why he included it. I can delete it though. 

> Would it be better to have a skipCARDTargetFeature flag and set it to True rather than completely remove it?

Since most of that block of code was copied and used for building the annotation data here, I think we can get rid of it from the feature method (we'll also have git history). Also, since I've had to make so many changes to how the CARD data is processed and wrangled, most of that block of code would have to change anyway if/when we do decide to include features again.